modules/portworx/scripts/install-portworx.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e -u
+set -o pipefail
+. vars
+
+PX_KVDB_DEVICE="$(echo "${PX_KVDB_DEVICE}" | tr '[:upper:]' '[:lower:]'|xargs)";
+
+if [[ "${PX_KVDB_DEVICE}" == "" ]]; then 
+  sed -i_sedtmp "/    kvdbDevice: XX_PX_KVDB_DEVICE_XX/d" "${vPX_Storage_Cluster_File}";
+elif [[ "${PX_KVDB_DEVICE}" == "auto" ]]; then 
+  sed -i_sedtmp "s,XX_PX_KVDB_DEVICE_XX,/dev/pwx_vg/pwxkvdb,g" "${vPX_Storage_Cluster_File}";
+else 
+  sed -i_sedtmp "s,XX_PX_KVDB_DEVICE_XX,${PX_KVDB_DEVICE},g" "${vPX_Storage_Cluster_File}";
+fi
+rm *_sedtmp
+
+vMasters="$("${kbCtl}" --kubeconfig="${vKubeConfig}" get node --selector='node-role.kubernetes.io/master' --no-headers=true -o custom-columns=":metadata.name")";
```

Comment:
`/master` role is deprecated, this will produce an empty $vMasters in new clusters, you may check `node-role.kubernetes.io/control-plane` instead

Reply:
Done